criu/hugetlb.c

```diff
@@ -0,0 +1,38 @@
+#include "hugetlb.h"
+#include "kerndat.h"
+
+struct htlb_info hugetlb_info[HUGETLB_MAX] = {
+	[HUGETLB_16KB] = { 16 * KB, MAP_HUGETLB_16KB },	   [HUGETLB_64KB] = { 64 * KB, MAP_HUGETLB_64KB },
+	[HUGETLB_512KB] = { 512 * KB, MAP_HUGETLB_512KB }, [HUGETLB_1MB] = { 1 * MB, MAP_HUGETLB_1MB },
+	[HUGETLB_2MB] = { 2 * MB, MAP_HUGETLB_2MB },	   [HUGETLB_8MB] = { 8 * MB, MAP_HUGETLB_8MB },
+	[HUGETLB_16MB] = { 16 * MB, MAP_HUGETLB_16MB },	   [HUGETLB_32MB] = { 32 * MB, MAP_HUGETLB_32MB },
+	[HUGETLB_256MB] = { 256 * MB, MAP_HUGETLB_256MB }, [HUGETLB_512MB] = { 512 * MB, MAP_HUGETLB_512MB },
+	[HUGETLB_1GB] = { 1 * GB, MAP_HUGETLB_1GB },	   [HUGETLB_2GB] = { 2 * GB, MAP_HUGETLB_2GB },
+	[HUGETLB_16GB] = { 16 * GB, MAP_HUGETLB_16GB },
```

Comment:
This format is because of the clang-format, I can't find any option to fix this.

What is your suggestion here? At first, I think of using macro to convert between size and flag. However, I will still need to keep a list of valid size and I want to iterate on that list like in `kerndat_get_hugetlb_dev` so I choose an easy solution to use an array like above.

Reply:
> This format is because of the clang-format, I can't find any option to fix this.

Oh, my... @adrianreber can we do something about this?

> What is your suggestion here?

Maybe I'm missing somthing but it seems to me that a single array  of valid sizes and a macro to convert a size to flag would suffice.

Reply:
See #1644 for possible workarounds

Reply:
> Maybe I'm missing somthing but it seems to me that a single array of valid sizes and a macro to convert a size to flag would suffice

I use the array of valid sizes with something like `arr[i]` so I think it's not an easy way to use macro here